### ISSUE TITLE
Game restarting after trying to persuade

### ISSUE THREAD

**Issue by igoramd** [ilpersi/BHBot#179]
The game is restarting after trying to persuade a familiar. It's happen when the new feature (auto persuade) is actived. The game acts faster then bot. Maybe any way to deactive AUTOBRIBE?

The logs:
[DungeonThread] AUTOBRIBE com.github.ilpersi.BHBot.BHBotLogger - Familiar encountered
[DungeonThread] WARN com.github.ilpersi.BHBot.BHBotLogger - Count for familiar squib is 0! Temporary removing it form the settings...
[DungeonThread] ERROR com.github.ilpersi.BHBot.BHBotLogger - Impossible to find the decline button, restarting...
[DungeonThread] WARN com.github.ilpersi.BHBot.BHBotLogger - Doing driver emergency restart...

**Desktop:**
 - OS: Windows 10 Home Single Language (x64)
 - Browser: Chromium
 - Version: BHBot v46.1 build on Wed Dec 31 21:00:00 GFT 1969 started.

[settings.zip](https://github.com/ilpersi/BHBot/files/5045384/settings.zip)

**Comment by ilpersi** [comment]
Try to comment out the `familiars` settings and to set `persuasionLevel` to 0. This should completely disable the auto-bribing and the auto-persuading features.

**Comment by igoramd** [comment]
I commented the `familiars` settings and set the `persuasionLevel` to 0 and the bot continued restarting the game :(

It looked like this:

`persuasionLevel 0`
`#bribeLevel 0`
`#familiars squib 0`
`#familiarScreenshot 1`
`#contributeFamiliars 1`

I also tried to comment on `persuasionLevel`, but got the same result. Is there any other setting for commenting?

settings.ini: 
[settings.zip](https://github.com/ilpersi/BHBot/files/5048413/settings.zip)

**Comment by ilpersi** [comment]
I am able to reproduce this and I am still thinking about the best way to fix this.

Probably, as the feature is built in the game, we do not need it anymore so more code refactoring is required. If you want to run the bot I suggest disabling the game feature while I try to find a better solution.

**Comment by igoramd** [comment]
Yes, I have currently disabled the game feature and am using the bot's AUTOBRIBE, but this makes a dungeon rush slower.

I'll be waiting for news. Thank you very much for your attention! <3 :)

**Comment by ilpersi** [comment]
I would prefer if we keep this open. It helps me remind things to do and it may be useful for other people having the same issue.

**Comment by ilpersi** [comment]
@tengen-toppa as one of the bot features is overlapping with a built-in one I am going to remove the unnecessary function from the bot otherwise it may be misleading for users.

**Comment by tengen-toppa** [comment]
> @tengen-toppa as one of the bot features is overlapping with a built-in one I am going to remove the unnecessary function from the bot otherwise it may be misleading for users.

Appreciate your hard work on BH bot.

**Comment by ilpersi** [comment]
@igoramd I just released the new [v46.4](https://github.com/ilpersi/BHBot/releases/tag/v46.4): it does not solve this issue, but it has many improvements and optimizations for encounter management that now is much faster than it used to be.

Check it out. 👍🏻

**Comment by igoramd** [comment]
Hey, it's incredibly faster now! It was really good.

I will have no more problems with that. Should I close this now?

**Comment by ilpersi** [comment]
I still need to make the bot work with built-in settings so l let's keep this open even if I consider it a low priority. :)
